docs/how_to/parse_time.rst

```diff
@@ -0,0 +1,105 @@
+.. _how_to_parse_times_with_parse_time:
+
+Parse times with `sunpy.time.parse_time`
+========================================
+
+.. code-block:: python
+
+    >>> import time
+    >>> from datetime import date, datetime
+    >>> import numpy as np
+    >>> import pandas
+    >>> from sunpy.time import parse_time
+
+The following examples show how to use `sunpy.time.parse_time` to parse various time formats, including both strings and objects, into an `astropy.time.Time` object(s).
+
+* strings:
+
+.. code-block:: python
```

Comment:
This combination of a list and code blocks renders pretty pants:

![image](https://user-images.githubusercontent.com/1391051/234614713-36abfac5-7cf8-4e6f-8066-48105d136f17.png)


I suggest we make these subheadings.

Reply:
Agreed but the subheadings look worse